### ISSUE TITLE
Legend for Diplomacy Overview

### ISSUE THREAD

**Issue by Caballero-Arepa** [yairm210/Unciv#10252]
It does not matter the color code used to indicate which color represents which diplomatic status, that conversation has already been done.

What matter is to tell the common player what that means.

So, as every diagram should, the Diplomacy Overview diagram should have a legend indicating the color code.

My concept is where I see here a blank space which could be used.
![250 sin título_20231005201303.png](https://github.com/yairm210/Unciv/assets/78449553/1050d299-c7c5-4d16-854e-d1981ddc6b36)
And for portrait:![250 sin título_20231005201307.png](https://github.com/yairm210/Unciv/assets/78449553/ef85b631-bc21-4923-9e49-cb07ae8bd349)

Tldr: Tell me what in the holy grail the colored lines mean.

**Comment by Caballero-Arepa** [comment]
> ¿como cada colombiano debe ser lleno de aguardiente? 😈
¡Esa es la vaina eeeehh jeje!

> ¿"Mostar"?
Uuhhh... you  saw  nothing . . .
